### DISCUSSION TITLE
Помогите разобраться почему не работает обновление дискорда при использовании hostlist

### DISCUSSION THREAD

**Discussion by AloofLibra** [bol-van/zapret#920] [Q&A] [answered]
Почему-то при использовании hostlist перестают работать обновления Discord. Просто вечно крутится попытка обновления при запуске. При переключении в None c той же стратегией обновления начинают работать. 

Роутер с OpenWRT, последняя версия Zapret, 50-Discord работает, сама стратегия рабочая (если открыть дискорд в вебе **или** отключить hostlist то всё будет работать), настроен DoT, в HostList добавлены все http адреса Discord из [треда](https://github.com/bol-van/zapret/discussions/475). 

По dns имени updates.discord.com бьются 4 ip адреса. Может в этом дело?.. Или я где-то ещё затупил?..

**Comment by numbereleven-a** [comment]
напиши полный конфиг дурения
за место ссылки на тред с адресами напиши адреса в твоём файле

с ходу можно MODE_FILTER поменять не на hostlist, а на autohostlist и тогда если проблема с не добелёнными доменами, они сами пропишутся 
врядли с dns, тогда у тебя и с none были бы проблемы

**Comment by numbereleven-a** [comment]
: <details><summary>мой лист дискорда</summary>
discord.com
discord-attachments-uploads-prd.storage.googleapis.com
dis.gd
discord.co
discord.design
discord.dev
discord.gg
discord.gift
discord.gifts
discord.media
discord.new
discord.store
discord.tools
discordapp.com
discordapp.net
discordmerch.com
discordpartygames.com
discord-activities.com
discordactivities.com
discordsays.com
comdiscord.com
discordapp.io
</details>

**Comment by AloofLibra** [comment]
Конфиг:

NFQWS_PORTS_TCP=80,443
NFQWS_PORTS_UDP=443,6969,50000-50099

NFQWS_TCP_PKT_OUT=$((6+$AUTOHOSTLIST_RETRANS_THRESHOLD))
NFQWS_TCP_PKT_IN=3
NFQWS_UDP_PKT_OUT=$((6+$AUTOHOSTLIST_RETRANS_THRESHOLD))
NFQWS_UDP_PKT_IN=0

NFQWS_OPT="
--filter-tcp=80 --dpi-desync=disorder2 <HOSTLIST> --new
--filter-tcp=443 --dpi-desync=disorder2 <HOSTLIST> --new
--filter-udp=6969 --dpi-desync=fake --dpi-desync-repeats=3 --dpi-desync-fake-quic=/opt/zapret/files/fake/quic_initial_www_google_com.bin <HOSTLIST> --new
--filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=3 --dpi-desync-fake-quic=/opt/zapret/files/fake/quic_initial_www_google_com.bin <HOSTLIST>
"

MODE_FILTER=autohostlist

Хостлист:

discordcdn.com
discordstatus.com
discord-attachments-uploads-prd.storage.googleapis.com
dis.gd
discord.co
discord.com
discord.design
discord.dev
discord.gg
discord.gift
discord.gifts
discord.media
discord.new
discord.store
discord.tools
discordapp.com
discordapp.net
discordmerch.com
discordpartygames.com
discord-activities.com
discordactivities.com
discordsays.com
updates.discord.com

**Reply by AloofLibra** [reply]
Автохостлист новых доменов дискорда не добавляет. Но точно работает т.к. домены других сайтов туда нормально приезжают

**Reply by numbereleven-a** [reply]
> i

попробуй с HOSTLIST

 --filter-tcp=80,443 --dpi-desync=disorder2 HOSTLIST  --new 

(только HOSTLIST в кавычки поставь, а то гитхаб сжирает их <хостлист>)
если ,6969,50000-50099 это для войса то смысла нет при использовании кастом скрипта,  перекинуть скрипт 50-discord из папки \init.d\*****\custom.d.examples в custom.d
PS
--filter-udp=443,6969 - тоже можно сократить в одну строку для красоты

**Reply by AloofLibra** [reply]
Там есть HOSTLIST, просто оно сожрано гитхабом))

**Reply by AloofLibra** [reply]
6969 - это для торрентов, 50000-50099 удалил, но это никак не повлияло

**Reply by numbereleven-a** [reply]
> 6969 - это для торрентов, 50000-50099 удалил, но это никак не повлияло

понял про 69 (а зачем?), про 5000 это просто как правильней типа, скрипт голоса будет отрабатываться когда уже обойдешь вход
попробуй другую страту чтоль
еще можешь дебаглог сделать (в варианте none и варианте autohostlist) и сюда скинуть  --filter-tcp=80,443 --debug=@/tmp/zapretlog.txt --dpi-desync=d...

**Reply by AloofLibra** [reply, marked as answer]
Проблема решилась обновлением OpenWRT и https-dns-proxy. Видимо всё же DNS-ы были кривые. 
Спасибо за помощь)
